Makefile

```diff
@@ -8,6 +8,7 @@ PROJECTS_PYCODESTYLE="sdk cli toolchain"
 BUILD_PARAMS=
```

Comment:
With current approach, I can't relogin under my second user account. I should clear the config file manually in this case. Is it expected?

Reply:
Will add logout